### PR TITLE
Make the sidebar accessible

### PR DESCRIPTION
**Pull request by Silbeg**
And not glaring!

Also fix registered events.

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -85,6 +86,7 @@ $unpaid: #B00100;
       -border-radius: 10px;
       border-radius: 10px;
       background-color: $paleGray;
+      color: $blue;
```
Comment (panpear) [inline app/assets/stylesheets/application.scss]: If this has the same color as the parent li, the color doesn't _have_ to be specified here.

File: app/views/layouts/_sidebar.html.erb [hunk 2]
```diff
@@ -6,11 +6,11 @@
   <li><%= link_to 'Home', welcome_index_path %></li>
 
   <!-- TODO - This should only display if a user has registered for events. The second part of this if statement doesn't work. -->
-  <% if user_signed_in? && !current_user&.user_events&.blank? %>
+  <% if user_signed_in? && current_user&.current_events&.present? %>
```
Comment (panpear) [inline app/views/layouts/_sidebar.html.erb]: Does present check for the existence of the array or it it contains items? I was having some issues with using 'present' to check on things before.

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (Silbeg) [inline app/assets/stylesheets/application.scss]: It actually seemed to make a difference, which is why I put it there.

File: app/views/layouts/_sidebar.html.erb (hunk 2 above)
Comment (Silbeg) [inline app/views/layouts/_sidebar.html.erb]: `present?` (the question mark is needed) checks to see that the array has elements. 

It is a rails only thing, though, so if you don't have rails it wouldn't work. 
It is better than using array&.empty?

**Pull request merged**